spec/services/google/parse_service_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::ParseService, type: :service do
+  describe '#call' do
+    context 'when parsing a page having 3 top ads and 1 bottom ad' do
```

Comment:
The test descriptions are too coupled with the fixtures making it hard to maintain. Prefer the following structure:

```
context 'given a page with top and bottom ads' do
  it 'retrieves the top ads', vcr: 'google/programming_courses_3_top_1_bottom' do

  end

  it 'retrieves the bottom ads', vcr: 'google/programming_courses_3_top_1_bottom' do

  end
end
```

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/9988cc8d24c0bff170a67f8cc38e7b31a6adf90d